c7n_kube/values.yaml

```diff
@@ -0,0 +1,8 @@
+action: warn
+certManager:
+  enabled: false
+port: 8080
+image:
+  repository: cloudcustodian/c7n-kube
```

Comment:
We currently don't publish `c7n-kube` images on its own, should we be using the https://hub.docker.com/r/cloudcustodian/c7n or are you expecting we'll create a slimmed down image that is *just* c7n-kube ?

Reply:
Oh, weird, I thought I had looked this up. I can swap that for the mono-image. There's some value in having slim images ... but not much.

Reply:
Ah, what I meant to say was, I created a c7n-kube image that we should start pushing on deploys. This would let us slim it down so people aren't installing something chonky (with potentially more security concerns) in their cluster.

Reply:
its mostly the generated junk in azure sdk that bloats the image, but yeah having a dedicated entry point for this use case does make sense. we've been having some flakey issues with cross arch docker image building in the main repo though.